flash-resume-server/server/controllers/experience.ts

```diff
@@ -0,0 +1,80 @@
+import { PrismaClient } from "@prisma/client";
+import { Request, Response } from "express";
+
+const prisma = new PrismaClient();
+
+const create = async (req: Request, res: Response) => {
+  try {
+    const newExperience = await prisma.experience.create({ data: req.body });
+    res.status(201).json(newExperience);
+  } catch (err) {
+    res.status(400).json({ message: err });
+  } finally {
+    await prisma.$disconnect();
+  }
+};
+
+const list = async (_: Request, res: Response) => {
```

Comment:
I don't know when we'll need this, but I think we should remove this once this gets deploy. This is okay for testing for right now

Reply:
Understood